### DISCUSSION TITLE
Importify?

### DISCUSSION THREAD

**Discussion by stepbaro** [watsonbox/exportify#144] [Ideas]
Would there be any possibility of incorporating a function to let you import back into Spotify to create new playlists? It would be great to be able to download all my likes (as an example), pull stuff out by genre or artist etc and then import back into a new playlist? Is this even possible?

Whatever, the tool is amazing - thanks for building :)

**Comment by watsonbox** [comment]
Thanks for the kind feedback.

I'm afraid it's not planned, no, but if anyone wanted to create a project to do this then it could be great. Whether that would be importing back into Spotify, or into some other music service would depend on use case.

If the idea got traction I'd be prepared to take a more serious look at [standard formats](https://github.com/watsonbox/exportify/issues/78) which would be easier to deal with in such a tool.
